ECI-1825 [OCI] Update docs with remediation steps when metrics are not being collected (#38441)
DataDog/documentation: content/en/integrations/guide/oci-integration-troubleshooting.md

**Note**: If you specified the subnet OCIDs in the optional configuration section, ensure that there is one subnet OCID per subscribed region. Do not make any other modifications to the existing stack before reapplying it.

## Metrics not being collected

Complete the following checks for each monitored region:

1. In the integration compartment, verify that the `dd-function-app` function application exists.
2. In `dd-function-app`, verify that the `dd-metrics-forwarder` function exists.

> 3. If you use custom subnets, verify that they meet the the permissions outlined [here][7]

Reviewer: I don't any permissions listed in that linked section. Is it the prerequisites you are referring to?

```suggestion
3. If you use custom subnets, verify that they meet these [permissions][7].
```

Author: Ah it is noted in the bolded note section after the steps. I will update to make this more clear

> 4. Check for stale Datadog service connector hubs. All Datadog created connector hubs will be in the format `dd-<service>-connectorhub-<suffix>`. If you find any, delete these Datadog-managed connector hubs.

Reviewer: ```suggestion
4. Check for stale Datadog service connector hubs. All Datadog created connector hubs are in the format: `dd-<service>-connectorhub-<suffix>`. Delete any stale Datadog-managed connector hubs.
```

**Note**: If you specified the subnet OCIDs in the optional configuration section, ensure that there is one subnet OCID per subscribed region. Do not make any other modifications to the existing stack before reapplying it.

## Metrics not being collected

Complete the following checks for each monitored region:

1. In the integration compartment, verify that the `dd-function-app` function application exists.
2. In `dd-function-app`, verify that the `dd-metrics-forwarder` function exists.
3. If you use custom subnets, verify that they meet the [permissions][7] (In the note after step 5).

> 4. Check for stale Datadog service connector hubs. All Datadog created connector hubs are in the format: `dd-<service>-connectorhub-<suffix>`. Delete any stale Datadog-managed connector hubs.

Reviewer 2: Rather than calling them "stale", maybe we could say confirm that the function target on connector hubs is the expected forwarding function, otherwise delete the connector hub? Customers won't know what "stale" means
